optaplanner-docs/src/modules/ROOT/pages/use-cases-and-examples/vaccination-scheduling/vaccination-scheduling.adoc

```diff
@@ -0,0 +1,266 @@
+[id="chap-optaplanner-vaccination_{context}"]
+= OptaPlanner Vaccination Scheduler
+
+:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+//:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
+
+ifdef::OPTAPLANNER-COMM[]
+
+:imagesdir: ../..
+
+:OPTAPLANNER: OptaPlanner
+:QUARKUS: Quarkus
+:OPTAPLANNER_JDK_VERSION: 11
+:OPTAPLANNER_MAVEN_VERSION: 3.6
+endif::OPTAPLANNER-COMM[]
+
+:context: optaplanner-vaccination-scheduler
+
+You can use the {OPTAPLANNER} vaccination appointment scheduler quick start to develop a vaccination schedule that is both efficient and fair. The vaccination appointment scheduler uses artificial intelligence (AI) to prioritize people and allocate time slots based on multiple constraints and priorities.
+
+.Prerequisites
+
+* OpenJDK {OPTAPLANNER_JDK_VERSION} or later is installed.
+
+
+ifdef::OPTAPLANNER-ENT[]
+Red Hat build of Open JDK is available from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required).
+endif::OPTAPLANNER-ENT[]
+
+
+* Apache Maven {OPTAPLANNER_MAVEN_VERSION} or higher is installed. Maven is available from the https://maven.apache.org/[Apache Maven Project] website.
+
+* An IDE, such as IntelliJ IDEA, VSCode, Eclipse, or NetBeans is available.
+
+
+ifdef::OPTAPLANNER-ENT[]
+* You have created a Quakus OptaPlanner project as described in xref:optaplanner-quarkus-con_getting-started-optaplanner[].
+endif::OPTAPLANNER-ENT[]
```

Comment:
In community this should link to
https://www.optaplanner.org/docs/optaplanner/latest/quickstart/quickstart.html#quarkusJavaQuickStart